frontend/src/pages/modelRegistrySettings/CreateModal.tsx

```diff
@@ -276,6 +278,22 @@ const CreateModal: React.FC<CreateModalProps> = ({ onClose, refresh, modelRegist
     hasContent(database) &&
     (!addSecureDB || (secureDBInfo.isValid && !configSecretsError));
 
+  if (!modelRegistryNamespace) {
+    return (
+      <ApplicationsPage
+        loaded
+        empty={false}
+        loadError={new Error('No registries namespace could be found')}
+        loadErrorPage={
+          <RedirectErrorState
+            title="Could not load component state"
+            errorMessage="No registries namespace could be found"
+          />
+        }
+      />
+    );
```

Comment:
Same rendering change here - we don't need `loadError` / `loadErrorPage` if we just pass the rendered error as children, it's kinda weird to not use the value you pass to `new Error()`.